mobile/types.go

```diff
@@ -264,8 +265,11 @@ func (tx *Transaction) GetHash() *Hash    { return &Hash{tx.tx.Hash()} }
 func (tx *Transaction) GetSigHash() *Hash { return &Hash{tx.tx.SigHash(types.HomesteadSigner{})} }
 func (tx *Transaction) GetCost() *BigInt  { return &BigInt{tx.tx.Cost()} }
 
-func (tx *Transaction) GetFrom() (address *Address, _ error) {
-	from, err := types.Sender(types.HomesteadSigner{}, tx.tx)
+func (tx *Transaction) GetFrom(chainID *big.Int) (address *Address, _ error) {
```

Comment:
Possibly even more limitations. CGO is one thing the mobile stuff needs to cross, but there's a ton of C and Java/ObjC code being generated, and sometimes a feature is missing here or there (e.g. you can't have uint.... because java).